### DISCUSSION TITLE
Recommendations for representing "global" events / observances / occasions

### DISCUSSION THREAD

**Discussion by BigglesZX** [schemaorg/schemaorg#2849] [General]
Firstly I hope this is the right place to put forward this question. If it's not, my apologies.

I administer an educational site that includes a calendar of global events, but these are not "Events" in the sense that Schema.org considers them – they don't have a location, and they are observed across a wide area (e.g. a country, an continent, or worldwide). They are more like global anniversaries, occasions or observances. Examples include Earth Day, Chinese New Year, World Book Day, International Women's Day and so on.

In our initial attempt at introducing microdata for these calendar pages, I have used the Event type, but I'm aware that as many – or most – of these events don't have a definable location (in the sense that Schema.org represents a location), and are not "attended" in a physical or digital sense, they cannot be valid.

Does anyone have any recommendations for correctly representing events like this, and/or is there a different or more abstract type that I should be using, rather than Event? Any thoughts or advice would be much appreciated.

**Comment by achristlex** [comment]
I'm still relatively new to the schema vocabulary, but I'd imagine that what you are looking for is [DayofWeek](https://schema.org/DayOfWeek) and specifically [PublicHolidays](https://schema.org/PublicHolidays)?

**Comment by thadguidry** [comment]
@BigglesZX Why do you think it would be "invalid"?  Is someone or some system telling you that use case is "invalid"?  Your use case explained sounds exactly like why we created the [Event](https://schema.org/Event) type to begin with.  The location property is entirely optional.  In the Schema.org vocabulary, all properties are optional, use 1 or many as you need.

Also, we have added [VirtualLocation](https://schema.org/VirtualLocation) to help during these pandemic times.
Scroll down on it and take a look at it's JSON-LD examples.  Does that help or make sense to answer your question better?

If your scenario is more like an Observance (Religious, Work Office, Team, whatever) we don't directly have that type.  So for now you can use [Event](https://schema.org/Event) or [EventSeries](https://schema.org/EventSeries) and multi-type or [additionalType](https://schema.org/additionalType) if need be, against whatever other vocabulary you want to use, such as Wikidata.

```json-ld
    <script type="application/ld+json">
    {
      "@context":  "https://schema.org/",
      "@id": "observance442",
      "@type": "Event",
      "additionalType": "https://www.wikidata.org/wiki/Q16675435"
      "name": "Our Green Work Out Day",
      "location": {
        "@type": "Place",
        "address": {
          "@type": "PostalAddress",
          "addressLocality": "Denver",
          "addressRegion": "CO",
          "postalCode": "80209",
          "streetAddress": "7 S. Broadway"
      },
      "startDate": "2013-09-14T21:30"
    }
    </script>
    ```

**Reply by BigglesZX** [reply]
This is helpful, thanks – we are now using VirtualLocation and the OnlineEventAttendanceMode.

I think since I wrote this post Google's Structured Data Testing Tool (which I was using for validation) has relaxed some of its mandatory/recommended logic, as our data now generates only a few warnings rather than the errors I was previously seeing.

**Reply by thadguidry** [reply]
Sure, see the above example I just wrote up for you.

**Reply by BigglesZX** [reply]
`additionalType` looks very useful for this as that is indeed the nature of most of our events. I'll get that added to our next release. Thanks again!
